src/components/menus/basicsMenu.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+  import MenuContents from '@src/components/menuContents.svelte';
+  import Input from '@src/components/input.svelte';
+  import { basicsStore, BasicsStore } from '@src/data/data';
+  let basics: BasicsStore = basicsStore;
```

Comment:
Do I really need to import the type if it can be applied by assigning `basicsStore` to the var, and thus remove the need to import `BasicsStore` to the file?